xcp_d/workflow/cifti.py

```diff
@@ -147,46 +147,68 @@ def init_ciftipostprocess_wf(
     %(correlations)s
     qc_file
         quality control files
-    """
-    workflow = Workflow(name=name)
-    workflow.__desc__ = f"""
-For each of the {num2words(n_runs)} CIFTI runs found per subject (across all
-tasks and sessions), the following post-processing was performed:
-"""
 
+    References
+    ----------
+    .. footbibliography::
+    """
     TR = get_cifti_tr(bold_file)
     if TR is None:
         metadata = layout.get_metadata(bold_file)
         TR = metadata['RepetitionTime']
+
     # Confounds file is necessary: ensure we can find it
     from xcp_d.utils.confounds import get_confounds_tsv
     try:
         confounds_tsv = get_confounds_tsv(bold_file)
     except Exception:
         raise Exception(f"Unable to find confounds file for {bold_file}.")
 
-    # TR = get_cifti_tr(cifti_file=bold_file)
+    workflow = Workflow(name=name)
+
+    filter_str = ""
+    if motion_filter_type:
+        if motion_filter_type == "notch":
+            filter_sub_str = (
+                f"band-stop filtered to remove signals between {band_stop_min} and "
+                f"{band_stop_max} breaths-per-minute using a notch filter, based on "
+                "@fair2020correction"
+            )
+        else:  # lp
+            filter_sub_str = (
+                f"low-pass filtered below {band_stop_min} breaths-per-minute, "
+                "based on @fair2020correction and @gratton2020removal"
+            )
+
+        filter_str = (
+            f"the six translation and rotation head motion traces were {filter_sub_str}. "
+            "Next, "
+        )
+
+    fd_str = (
+        f"{filter_str}framewise displacement was calculated using the formula from "
+        f"@power_fd_dvars, with a head radius of {head_radius} mm"
+    )
+
+    dummytime_str = ""
     initial_volumes_to_drop = 0
     if dummytime > 0:
-        initial_volumes_to_drop = int(np.floor(dummytime / TR))
-        workflow.__desc__ = workflow.__desc__ + f""" \
-before nuisance regression and filtering of the data,  the first
-{num2words(initial_volumes_to_drop)} were discarded.
-Both the nuisance regressors and volumes were demean and detrended. Furthermore, any volumes
-with framewise-displacement greater than {fd_thresh} mm [@power_fd_dvars;@satterthwaite_2013] were
-flagged as outliers and excluded from nuisance regression.
-"""
-
-    else:
-        workflow.__desc__ = workflow.__desc__ + f""" \
-before nuisance regression and filtering,both the nuisance regressors and volumes were demeaned
-and detrended. Volumes with framewise-displacement greater than {fd_thresh} mm
+        initial_volumes_to_drop = int(np.ceil(dummytime / TR))
```

Comment:
This is changed from `int(np.floor(dummytime / TR))` to match the nifti workflow's approach. Does anyone know if there's a reason the cifti workflow uses `numpy.floor`, while the nifti workflow uses `numpy.ceil`?

Reply:
definitely a typo - they should both be ceil